lib/elixir/lib/inspect/algebra.ex

```diff
@@ -31,6 +31,11 @@ defmodule Inspect.Opts do
       options. Useful when implementing the `Inspect` protocol for nested structs
       to pass the custom options through.
 
+      It supports some pre-existing keys:
```

Comment:
Nitpick: maybe this?

```suggestion
      It supports some pre-defined keys:
```